dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -199,4 +199,4 @@ set400SCOnTooManyParentDirs.name=Set 400 for invalid path traversal request
 set400SCOnTooManyParentDirs.desc=When this property is set to true, a request with an invalid path traversal is returned with a 400 status code instead of 500. The default is false. You can use this property with servlet-5.0 and earlier. For servlet-6.0 and later, the 400 response code is returned by default and this property has no effect.
 
 allowAbsoluteFileNameForPartWrite.name=Write part files to absolute file locations
-allowAbsoluteFileNameForPartWrite.desc=When this property is set to true, the runtime treats the filename argument on the part.write method call as an absolute location, rather than saving the filename arguement under the temporary location. In servlet-6.0 and earlier, the default value is false. In servlet-6.1 and later, the default value is true.
+allowAbsoluteFileNameForPartWrite.desc=When this property is set to true, the runtime treats the filename argument on the part.write method call as an absolute location, rather than saving the filename argument under the temporary location. In servlet-6.0 and earlier, the default value is false. In servlet-6.1 and later, the default value is true.
```

Comment:
```suggestion
allowAbsoluteFileNameForPartWrite.desc=When this property is set to true, the runtime treats the filename argument on the part.write method call as an absolute location, rather than saving the filename argument under the temporary location. In servlet-6.1 and later, the default value is true. In servlet-6.0 and earlier, the default value is false. 
```